### DISCUSSION TITLE
使用最新的 3.0.1 版本的 flink-cdc, debezium 可以升级到最新的 2.6 版本么？

### DISCUSSION THREAD

**Discussion by stardustman** [apache/flink-cdc#3252] [Q&A]
如果不能那可以使用哪一个接近最新版本的 debezium?

**Comment by stardustman** [comment]
😄，就你们这种项目文档的水平，还想推广让人使用，做梦去吧。

**Comment by yuxiqian** [comment]
Hello @stardustman, Flink CDC is compatible with Debezium is 1.9.8 Final and lower. We can't upgrade to Debezium 2.x since it requires Java 11+. Since Flink will not drop Java 8 support until 2.0 Release, Flink CDC has no choice but stick with current Debezium version. If you're willing to contribute on this, feel free to open a PR :)
